sda-commons-server-morphia/README.md

```diff
@@ -97,10 +97,11 @@ mongo:
   username: "${MONGODB_USERNAME:-}"
   password: "${MONGODB_PASSWORD:-}"
   useSsl: ${MONGODB_USE_SSL:-true}
-  caCertificate: "${MONGODB_CA_CERTIFICATE:-}"
+  
+caCertificate:
+  customCaCertificateDir: "${CA_CERTIFICATE_DIR:-}"
```

Comment:
By default we do not have this configuration for MongoDB. But we use the default configuration of the CaCertificateBundle. So we should document the default directory where certificates can be mounted and that this approach is preferred over setting the caCertificate in the environment.